static/app/components/profiling/flamegraphSearch.tsx

```diff
@@ -321,13 +327,45 @@ function FlamegraphSearch({
     <StyledSearchBar
       size="xs"
       placeholder={t('Find Frames')}
-      query={search.query}
+      defaultQuery={search.query}
```

Comment:
just delegate the input value rendering to the `SearchBar` - app search state can remain in `flamegraphSearch`